### DISCUSSION TITLE
vercel.app can be used normally, but now.sh does not seem to be restored in China

### DISCUSSION THREAD

**Discussion by xhemj** [vercel/vercel#6241] [Help] [answered]
https://www.vercel-status.com/incidents/r758bhbklgfd

**Comment by fu-sen** [comment, marked as answer]
All references to `.now.sh` are now being redirected to` .vercel.app`. Therefore, dealing with `.now.sh` will be a low priority.
https://vercel.com/changelog/urls-are-becoming-consistent

**Reply by xhemj** [reply]
I mean in China ` .now.sh` cannot redirected to ` .vercel.app`
